vin/app/models/concerns/package.rb

```diff
@@ -1,18 +0,0 @@
-#enconding: utf-8
-
-class Package < ActiveRecord::Base
-  # Attributes
-  attr_accessible :wine1_id, :wine2_id, :wine3_id, :wine4_id,
-                  :wine5_id, :wine6_id, :type, :note
```

Comment:
I have to delete all the `concerns` files, when i was playing with the rails console i decided to put the old-raw-versions of the model there as a backup... Shame on me for the bad model and for commiting this! hahaha